purchases-sample/src/main/res/layout/fragment_offering.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:bind="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <variable
+                name="offering"
+                type="com.revenuecat.purchases.Offering" />
+
+    </data>
+
+    <androidx.core.widget.NestedScrollView
+            android:id="@+id/offering_fragment_scroll_view"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:clipToPadding="false"
+            android:fillViewport="true">
+
+        <com.google.android.material.card.MaterialCardView
+                android:id="@+id/offering_details_card"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/padding_tiny"
+                android:transitionName="@string/offering_fragment_transition">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:padding="@dimen/padding_small">
+
+                <androidx.appcompat.widget.AppCompatTextView
```

Comment:
This confused me a bit. I think I would add the offering identifier here and have the description under. Basically switch description with identifier here:

<img width="368" alt="Screen Shot 2021-04-02 at 3 13 11 PM" src="https://user-images.githubusercontent.com/664544/113457921-f26cf780-93c5-11eb-9e30-c123628459bf.png">

So it reads:

default
description:
Default set